### ISSUE TITLE
Missing ARIA Landmark role="search"

### ISSUE THREAD

**Issue by davidstenger** [18F/web-design-standards#1654]
**Issue**:
The page is missing an ARIA landmark role of "search" for the Search area(s) of the page. Landmark roles assist screen reader users by conveying where they are on the page and can be used as an alternate way to navigate the page.

**Solution**:
Add ARIA role="search" for the Search regions of the page. The Search role must surround the visible label, the search field, and the search button.

Refer to: http://www.w3.org/TR/wai-aria/roles#search

Example:
```
<form action="…">
  <div role="search">
    <input type="search" aria-label="Search"> <button type="submit">Search</button>
  </div>
</form>
```

**Comment by donjo** [comment]
Hey @davidstenger do you happen to remember which search fields didn't have the search role applied? I just combed through the search examples I could find and they seemed to have `<div role="search">` applied. It's possible that this had been fixed at some point without referencing this issue but just checking to make sure.
